### DISCUSSION TITLE
How do i import Custom UTAU Voicebanks

### DISCUSSION THREAD

**Discussion by zeroninefourfour** [stakira/OpenUtau#474] [Q&A]
Uhhh,How do i import it?Import the folder or import what?It doesnt let me choose a folder. or UTAU voicebanks and OpenUTAU Voicebank are not same thing??

**Comment by zeroninefourfour** [comment]
and ii cant choose .wav files. 🤔

**Comment by oxygen-dioxide** [comment]
Here is the recommended way to import your voicebank: Compress it into a .zip archive file first, then install it in OpenUTAU. 

Besides, if your voicebank only contains ASCII filenames, you can also directly copy it to your OpenUTAU's Singer folder.
